article-bitbucket-pull-requests/article-bitbucket-pull-requests.md

```diff
@@ -0,0 +1,172 @@
+
+##### Git command line example
+This is a simple Git example of the procedure for pulling changes made by another user from a fork of a Bitbucket repository, back into the original repository also on Bitbucket.
+Check for incoming changes *(one change detected)*.
+```
+$ git fetch && git log ..origin/master
+  Password:
+  commit 2f41d64e75f2b4f0405cbfb1d2f882882809c209
+  Author: ANDREW <alui@atlassian.com>
+  Date: Thu Sep 8 11:50:08 2011 +1000
+  adding images
+```
+
+Check for outgoing changes (*no changes detected*).
+```
+$ git fetch && git log origin/master..
+  Password:
+  remote: Counting objects: 6, done.
+  remote: Compressing objects: 100% (3/3), done.
+  remote: Total 4 (delta 1), reused 0 (delta 0)
+  Unpacking objects: 100% (4/4), done.
+  From https://staging.bitbucket.org/alui/alui-git
+   2f41d64..7e28ddb master -> origin/master
+  Pull the changes into the local repository.
+```
+
+```
+$ git pull
+  Password:
+  Updating 2f41d64..7e28ddb
+  Fast-forward
+    Thumbs.db | Bin 0 -> 11776 bytes
+    file.txt | 4 +++-
+    2 files changed, 3 insertions(+), 1 deletions(-)
+    create mode 100644 Thumbs.db
+```
+
+## Check the build status on a pull request
+If your vendor or build system runs a build when you make a commit, you can see the status of the build(s) on each commit.  
+After you create a pull request, its **Overview** tab includes the build status for all the commits on that pull request. Click the **Commits** tab to see the builds statuses separated by commits.
+-- Image goes here build_status_pr
+The **Builds** column contains one of these statuses:
+ - appears if all builds for that commit passed
+ - appears if no builds for that commit failed and at least one build is in progress
+ - appears if at least one build for that commit failed
+
+From the **Commit** column, click the link to the commit that you want to see. On the right side of the commit page, you see the build status icon again and a link to more details. Click the link to see details of each build for the commit. The table that opens when you click a build status includes only the most recent status for each build, not a history of builds for the commit.  
+ -- Image goes here build_status_commit  
+You can also check the build statuses for a repository without creating a pull request. To see the build status on a commit, press **Commits** on the left side of the repository.
+
+<sub><sup>This article is taken almost directly from (Atlassian's Working with pull requests)[https://confluence.atlassian.com/bitbucket/work-with-pull-requests-223220593.html]. Some modifications were made to the layout.</sup></sub>
```

Comment:
There aren't equivalent tags in Markdown. It rendered and made it small so that it was more like a footer than a header. Its part of the license that they use. I don't mind where it goes.